### PR TITLE
fix: allow normalised experiments css value to be false

### LINKED ISSUE #15742: `experiments.css` cannot be disabled when `experiments.futureDefaults` is enabled
# Bug report

**What is the current behavior?**

When `experiements.futureDefaults` is enabled but `experiments.css` is disabled, webpack fails with the following error.

```
Error: Conflict: Multiple chunks emit assets to the same filename main.css
```

I believe this behaviour was introduced in `5.67.0` via (#15200). Specifically...
https://github.com/webpack/webpack/blob/1489b91a98daf7e59207da7d3b29ad12750a9ecf/lib/config/normalization.js#L185

When `experiments.css` is set to `false`, its value is normalised as `undefined`. So the `experiments.futureDefaults` value is used in its place here...
https://github.com/webpack/webpack/blob/1489b91a98daf7e59207da7d3b29ad12750a9ecf/lib/config/defaults.js#L164

**If the current behavior is a bug, please provide the steps to reproduce.**

`webpack.config.js`

```javascript
{
  experiments: {
    css: false,
    futureDefaults: true,
  },
}
```

**What is the expected behavior?**

webpack should build successfully with `experiments.css` disabled.

I would like to be able to enable `experiments.futureDefaults` so that I can ensure compatibility with the next major webpack version. But for some experiments, like `experiments.css`, I would like to be able opt-out for now until I better understand the upgrade path.

**Other relevant information:**
webpack version:  5.72.0
Node.js version: 16.14.2
Operating System: Ubuntu

### PR DESCRIPTION
**Pull request by jdanil**
Fixes #15742.

**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**

Yes.

**Does this PR introduce a breaking change?**

No.

**What needs to be documented once your changes are merged?**

Nothing.

### CODE REVIEW

File: declarations/WebpackOptions.d.ts [hunk 1]
```diff
@@ -3498,11 +3498,11 @@ export interface ExperimentsNormalizedExtra {
 	/**
 	 * Enable css support.
 	 */
-	css?: CssExperimentOptions;
+	css?: false | CssExperimentOptions;
 	/**
 	 * Compile entrypoints and import()s only when they are accessed.
 	 */
-	lazyCompilation?: LazyCompilationOptions;
+	lazyCompilation?: false | LazyCompilationOptions;
```
Comment (vankop) [inline declarations/WebpackOptions.d.ts:3505]: `buildHttp` should not have `false |` since there is no way to pass boolean in options

**Comment by jdanil** [comment]
@vankop thanks for the additional commit. Is there anything else you need from me to progress this?

**Comment by jdanil** [comment]
cc: @sokra.

**Review by sokra — APPROVED** [review]

**Comment by sokra** [comment]
Thanks

**Pull request merged**